src/ui/MLFacet/MLFacet.ts

```diff
@@ -329,6 +329,19 @@ export class MLFacet extends Component {
     this.updateAppearance();
   }
 
+  /**
+   * Sets a flag indicating whether the facet values should be returned in their current order.
+   *
+   * Setting the flag to true helps ensuring that the values do not move around while the end-user is interacting with them.
+   *
+   * The flag is automatically set back to false after a query is built.
```

Comment:
I think it would be better to reset "after the query is performed and cannot be modified by any other piece of code in the UI" 

For example, it would only reset on each "duringQuery" event.

It seems to make more sense that way, as you want to put the flag for a "query", and any additional or custom code that would need to interact with the query further down the line in the UI would have to correct value for the flag, as it was sent to the search API.